db/orders.js

```diff
@@ -0,0 +1,158 @@
+const client = require("./client");
+
+// Does not return order's products
+async function getOrderById (id) {
+    try{
+        const {rows: [order]} = await client.query(`
+            SELECT o.*,  p.name
+            FROM orders AS o
+            INNER JOIN order_products AS op ON op."orderId" = o.id
+            INNER JOIN products AS p ON op."productId" = p.id;
+            WHERE id=$1
+        `, [id]);
+        return order;
+    } catch (error) {
+        throw error;
+    };
+};
+
+// Does not return order's products
+async function getAllOrders() {
+    try {
+        const {rows} = await client.query(`
+            SELECT o.*,  p.name
+            FROM orders AS o
+            INNER JOIN order_products AS op ON op."orderId" = o.id
+            INNER JOIN products AS p ON op."productId" = p.id;
+        `);
+        return rows;
+    } catch (error) {
+        throw error;
+    };
+};
+
+// Does not return order's products
+async function getOrdersByUser({id}) {
```

Comment:
let's get rid of the curlies and define this with `getOrdersByUser(user_id)`

Reply:
Following the milestone guidelines.